chromadb/api/types.py

```diff
@@ -135,6 +112,43 @@ class IncludeEnum(str, Enum):
     data = "data"
 
 
+# Record set
+class RecordSet(TypedDict):
+    ids: IDs
+    embeddings: Optional[Embeddings]
+    metadatas: Optional[Metadatas]
+    documents: Optional[Documents]
+    images: Optional[Images]
+    uris: Optional[URIs]
+
+
+def does_record_set_contain_data(record_set: RecordSet, include: List[str]) -> bool:
+    if len(include) == 0:
+        raise ValueError("Expected include to be a non-empty list")
+
+    error_messages = []
+    for key in include:
+        if key not in record_set:
+            error_messages.append(
+                f"Expected include key to be a a known field of RecordSet, got {key}"
+            )
+
+    if len(error_messages) > 0:
+        raise ValueError(", ".join(error_messages))
+
+    for key, value in record_set.items():
+        if key not in include:
+            continue
+
+        if isinstance(value, list):
+            if len(value) == 0:
+                raise ValueError(f"Expected {key} to be a non-empty list")
+
+            return True
```

Comment:
This will return `True` if only one of the keys in `include` is present, but not all of them.

Reply:
this is intended for the use case. updated the name to be `does_record_set_contain_any_data` to reflect the intention

Reply:
Can we add a comment that that's what this does?

Reply:
sure